### ISSUE TITLE
Remove deprecated mediators from documentation samples 

### ISSUE THREAD

**Issue by abeykoon** [wso2/product-ei#3927]
**Description:**
We removed samples with deprecated mediators mentioned at [1] by PR [2]. Following samples are removed.

1. synapse_sample_157.xml
2. synapse_sample_460.xml
3. synapse_sample_470.xml
4. synapse_sample_652.xml

As a result following doc pages **(6.5.0 version of them)** needs to be updated/removed

NOTE: <In> and <out> mediators are used in a lot of samples. Hence, we will not update them in EI 6.5.0 distribution. 

**Main mediator page**
https://docs.wso2.com/display/EI640/ESB+Mediators

**Priority Executors + Enqueue Mediator**
https://docs.wso2.com/display/EI640/Sample+652%3A+Priority+Based+Message+Mediation
https://docs.wso2.com/display/EI640/Enqueue+Mediator
https://docs.wso2.com/display/EI640/Prioritizing+Messages
https://docs.wso2.com/display/EI640/Adding+a+Priority+Executor
https://docs.wso2.com/display/EI640/Deleting+a+Priority+Executor
https://docs.wso2.com/display/EI640/Editing+a+Priority+Executor
https://docs.wso2.com/display/EI640/Enqueue+Mediator

**POJO Command Mediator**
https://docs.wso2.com/display/EI640/POJOCommand+Mediator

**Bean Mediator + Spring Mediator**
https://docs.wso2.com/display/EI640/Bean+Mediator
https://docs.wso2.com/display/EI640/Sample+470%3A+How+to+Initialize+and+use+a+Spring+Bean+as+a+Mediator
https://docs.wso2.com/display/EI640/Spring+Mediator
https://docs.wso2.com/display/EI640/Mediating+with+Spring
https://docs.wso2.com/display/EI640/EJB+Mediator

**Conditional Router Mediator**
https://docs.wso2.com/display/EI640/Conditional+Router+Mediator
https://docs.wso2.com/display/EI640/Router+Mediator
https://docs.wso2.com/display/EI640/Sample+157%3A+Conditional+Router+for+Routing+Messages+based+on+HTTP+URL%2C+HTTP+Headers+and+Query+Parameters

**Event Mediator**
https://docs.wso2.com/display/EI640/Event+Mediator
https://docs.wso2.com/display/EI640/Publish+Event+Mediator
https://docs.wso2.com/display/EI640/Sample+460%3A+Introduction+to+Eventing+and+Event+Mediator
https://docs.wso2.com/display/EI640/Working+with+Topics+and+Events

**Comment by samgnaniah** [comment]
@isudana @abeykoon We have added notes for all these in the documentation. However, it looks like the samples have been removed completely from the distribution. Should we instead be deleting these pages in the docs? 

Normally we just put a note saying these are deprecated, however, it's unusual that the samples are removed from the distribution and exist in the docs.

Shall we remove these samples from the docs as well?
